api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -1273,7 +1270,18 @@ public static void stopModule(Module mod, boolean isShuttingDown) {
 		
 		return dependentModulesStopped;
 	}
-	
+
+	/**
+	 * Checks if the dependentModule's requirements are in the same package as the module in-check
```

Comment:
You could change this to: Checks if a module is required by another
dependentModule: the module whose required modules we are to check
modulePackage: the package of the module to check if required by another
return: true if the module is required, else false